plugins/api-docs/README.md

```diff
@@ -218,6 +219,27 @@ security:
     - [read_pets, write_pets]
 ```
 
+### gRPC rendering protoc-gen-doc descriptors
+
+To configure the gRPC api widget to render your generated [protoc-gen-doc](https://github.com/pseudomuto/protoc-gen-doc) files add the following annotation to your api entities: `'backstage.io/definition-generated-by': 'protoc-gen-doc'`
```

Comment:
In theory, there could be other plugins similar to `GenDocu` using the same `protoc-gen-doc` output. Perhaps the annotation should be `gendocu-com/grpc-docs`? Also, that plugin may very well decide to use a different input in the future and not rely on `protoc-gen-docs` at all...

Reply:
Thanks, valid points, I am not familiar with this whole ecosystem, and tbh thought they are the same :D But in this case it makes much more sense to have the annotation specify the plugin that consumes the`protoc-gen-doc` output